src/pages/ReportDetailsPage.tsx

```diff
@@ -196,10 +196,27 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
             });
         }
 
+        if ((isGroupChat || (isChatRoom && ReportUtils.canLeaveChat(report, policy ?? null))) && !isThread) {
```

Comment:
Agree with you at this point `&& !isThread` if it is thread we should display leave row

Reply:
Agree, but sort of out of scope for the original ticket maybe. I believe there is a separate project to make this change.